run/local-cluster.sh

```diff
@@ -105,7 +104,7 @@ for i in $(seq 1 $NODES); do
     CMD="init"
   fi
   # Command args specify two data directories per instance to simulate two physical devices.
-  CMD_ARGS="-gossip=${HOSTS[1]}:$RPC_PORT -stores=hdd=/tmp/disk1,hdd=/tmp/disk2 -rpc=${HOSTS[$i]}:$RPC_PORT -http=${HOSTS[$i]}:$HTTP_PORT"
+  CMD_ARGS="-gossip=${HOSTS[1]}:$HTTP_PORT -stores=hdd=/tmp/disk1,hdd=/tmp/disk2 -http=${HOSTS[$i]}:$HTTP_PORT"
```

Comment:
Maybe we should change -http to -addr. -addr is what's used from clients as well. Would be nice to just have a single argument.